gulpfile.js

```diff
@@ -164,18 +164,19 @@ function getTmpJsPath(locale) {
  * @return {!Array<string>} The list of generated task names.
  */
 function repeatTaskForAllLocales(taskName, dependencies, operation) {
-  const tasks = [];
-  ALL_LOCALES.forEach((locale) => {
+  return ALL_LOCALES.map((locale) => {
     // Convert build-js-$ to build-js-fr, for example.
     const replaceTokens = (name) => name.replace(/\$/g, locale);
-
     const localeTaskName = replaceTokens(taskName);
-    tasks.push(localeTaskName);
     const localeDependencies = dependencies.map(replaceTokens);
 
-    gulp.task(localeTaskName, localeDependencies, () => operation(locale));
+    gulp.task(localeTaskName, gulp.series(
+      gulp.parallel(...localeDependencies),
+      () => operation(locale),
```

Comment:
Our style guide uses 4 spaces when listing parameters after a new line:
```
func(
    a,
    b);
```
Please update here and below.